dvc.yaml

```diff
@@ -1,37 +1,37 @@
 stages:
   recommend-mind-val:
-    cmd: python -m poprox_recommender.evaluation.generate -M MINDlarge_dev -o outputs/mind-val-recommendations.parquet
+    cmd: python -m poprox_recommender.evaluation.generate -M MINDlarge_dev -o outputs/mind-val/
     deps:
       - src/poprox_recommender/evaluation/generate.py
       - data/MINDlarge_dev.zip
     outs:
-      - outputs/mind-val-recommendations.parquet
+      - outputs/mind-val/recommendations.parquet
 
   measure-mind-val:
     cmd: python -m poprox_recommender.evaluation.evaluate -M MINDlarge_dev mind-val
     deps:
       - src/poprox_recommender/evaluation/evaluate.py
       - data/MINDlarge_dev.zip
-      - outputs/mind-val-recommendations.parquet
+      - outputs/mind-val/recommendations.parquet
     outs:
       - outputs/mind-val-user-metrics.csv.gz
     metrics:
       - outputs/mind-val-metrics.csv:
           cache: false
```

Comment:
Do we want to move the metric output (both declared here and in the `evaluate.py` code) into the run's directory? (do we want to do that as a later PR?)

Reply:
“Do we” is a genuine question here — it's fine to merge this and decide we want to move the metric outputs later.

Reply:
That sounds like a good idea